projects/aca-content/src/lib/components/search/search-save/dialog/save-search-dialog.component.html

```diff
@@ -0,0 +1,53 @@
+<h2 mat-dialog-title>{{"APP.BROWSE.SEARCH.SAVE_SEARCH.MODAL_HEADER" | translate}}</h2>
+
+<mat-dialog-content>
+  <form [formGroup]="form" (submit)="submit()">
+    <mat-form-field class="aca-save-search-dialog__form-field">
+      <mat-label>{{ 'APP.BROWSE.SEARCH.SAVE_SEARCH.NAME_LABEL' | translate }}</mat-label>
+      <input
+        id="aca-save-search-dialog-name-input"
+        [attr.aria-label]="'APP.BROWSE.SEARCH.SAVE_SEARCH.NAME_LABEL' | translate"
+        matInput
+        required
+        [formControlName]="'name'"
+        adf-auto-focus
+      />
+
+      <mat-error *ngIf="form.controls['name'].touched">
+                <span *ngIf="form.controls['name'].errors?.required">
+                    {{ 'APP.BROWSE.SEARCH.SAVE_SEARCH.NAME_REQUIRED_ERROR' | translate }}
+                </span>
+                <span *ngIf="!form.controls['name'].errors?.required && form.controls['name'].errors?.message">
+                    {{ form.controls['name'].errors?.message | translate }}
```

Comment:
Are we sure the error messages will get translated in other languages like this?

Reply:
yes, they will as that's the return object of forbidOnluSpaces: 
{
            message: `NODE_FROM_TEMPLATE.FORM.ERRORS.ONLY_SPACES`
 }